indexing-hadoop/src/main/java/io/druid/indexer/IndexGeneratorJob.java

```diff
@@ -214,6 +218,9 @@ public boolean run()
 
   public static class IndexGeneratorMapper extends HadoopDruidIndexerMapper<BytesWritable, Text>
   {
+    final QueryGranularity rollupGranularity = getConfig().getGranularitySpec().getQueryGranularity();
```

Comment:
I don't think getConfig() will be able to return anything here. It is only filled in after hadoop calls "setup".